Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -96,6 +96,8 @@ static void webkit_user_content_manager_class_init(WebKitUserContentManagerClass
      * webkit_user_content_manager_register_script_message_handler()
      *
      * Since: 2.8
+     *
+     * Deprecated: 2.40
```

Comment:
I don't think we want to deprecate this signal, this is not a special case of the other when there's no reply, because with the other we always end up creating a reply and sending a undefined value when the script doesn't really expect any reply.